### ISSUE TITLE
Creature Light Sources

### ISSUE THREAD

**Issue by Djeryv** [ClassicUO/ClassicUO#428]
Although light sources seem to work, the creatures (like fire elementals) don't have their glow when they are equipped with their light source.

Windows 10
64 Bit
UO Client 7.0.15.1

**Comment by Djeryv** [comment]
To also add to this, if my character has a torch or lantern...that works fine with the light.

**Comment by andreakarasho** [comment]
Because we dont consider a body as a lightsource,  but equipment only like the torch

**Comment by andreakarasho** [comment]
probably this issue is related to https://github.com/andreakarasho/ClassicUO/issues/353

**Comment by Djeryv** [comment]
I don't think it is related since you fixed #353 and the latest dev doesn't fix this issue.

When we add light sources to monsters, does it get put in Layer Invalid or perhaps First Valid Layer?

**Comment by Djeryv** [comment]
On RunUO servers, the light source is an item that gets equipped on a mobile to make them glow. Unlike torches and lanterns, light sources have no appearance to them. They just glow. I tried adding this light source to different layers of the creature with no success. If I place an item (called LightSource in RunUO) on the ground...it glows in ClassicUO. If I equip it (even on a human NPC) it does not glow like a lantern does. I am not sure if anything of this helps with the issue.

**Comment by Djeryv** [comment]
Here is the Fiddler info for a wearable light, perhaps it can search for the tiledata?

![image](https://user-images.githubusercontent.com/49735284/64823830-14055480-d57e-11e9-8dc2-46a241e238c1.png)

**Comment by andreakarasho** [comment]
done, thank you :)

**Comment by Djeryv** [comment]
Can you also give the human/elf bodies the ability to show this type of light? They currently do not. So for example I have a human ghost that glows and they are equipped with the same light source as the fire ellys.

**Comment by andreakarasho** [comment]
Odd.
It should work for human too

**Comment by Djeryv** [comment]
Let me try again to be absolutely sure. Give me a few minutes.

**Comment by Djeryv** [comment]
Yeah...these human ghosts have the same light source equipped as the fire elly in the pick. Isn't working for some reason.

![image](https://user-images.githubusercontent.com/49735284/64911513-3d41f400-d6e8-11e9-9bd0-68c24d16bfb3.png)

**Comment by andreakarasho** [comment]
Ghost = dead?

**Comment by Djeryv** [comment]
No...just a creature that is set to hue of white and their clothes are white to match.

Sorry for the confusion.

**Comment by andreakarasho** [comment]
Did a fix, unfortunately i can't test.
Let me know please

**Comment by Djeryv** [comment]
Give me a few minutes to compile.

**Comment by andreakarasho** [comment]
fixed
